documentation/docs/30-advanced/20-hooks.md

```diff
@@ -86,6 +86,8 @@ export async function handle({ event, resolve }) {
 }
 ```
 
+If an error is thrown during `handle`, it's treated as a fatal error. Depending on the requested response type either a JSON error response or the static fallback `src/error.html` is returned.
```

Comment:
More evidence that we need a dedicated error handling page — we talk about `src/error.html` but it isn't included in the templates, and we don't explain what can go in it

```suggestion
If an error is thrown during `handle`, it is treated as fatal, and SvelteKit will respond with a JSON representation of the error or a fallback error page — which can be customised via `src/error.html` — depending on the `Accept` header. (Note that `resolve(...)` will never throw an error, it will always return a `Promise<Response>` with the appropriate status code).
```